test/utils.js

```diff
@@ -7,7 +8,7 @@ function doPost(endpoint, method, body, callback, forceQuerystring) {
 
     if (body) {
         if (useQuerystring) {
-            req.query(body);
+            req.query(qs.stringify(body));
```

Comment:
This definitely resolves the parsing of the querystring in the qs format, however I did notice that the logging of this message no longer displays, all you see is the base url without the parameters.
ex) VERBOSE_HTTP logging will only display : 
HTTP  POST    200 http://chase-hutchens.cs261.net/api/v1/inventory/hu9iixc387duz0k9 (99ms)
instead of :
HTTP  POST    200 http://chase-hutchens.cs261.net/api/v1/users/383/inventory/create?items%5B0%5D%5Bitemid%5D=254&items%5B0%5D%5Bquantity%5D=5&items%5B15D%5Bshortname%5D=INVENTORY8CFD&_session=2AbsD0JwoDJEgSnv4EeGf7HSg6q1m-j8&_token=w0eai94axhnz5mi (92ms)